docs/docs/deploy/deploy-rasa-plus.mdx

```diff
@@ -86,6 +53,63 @@ kubectl --namespace <your namespace> \
 oc --namespace <your namespace> \
     create secret docker-registry rasa-plus-pull-secret \
     --docker-server=gcr.io \
+    --docker-email=test-customer@rasa-releases.iam.gserviceaccount.com \
     --docker-username=_json_key \
     --docker-password="$(cat <your-json-registry-creds-file>.json)"
 ```
+
+## **Step 2: securely store the license key in a secret**
+
+Step 1: Save the license string to a file, for example:
+```sh
+echo "rasaplus-license-key-you-were-emailed" > license.txt
+```
+
+Step 2: Create a secret to hold the license key:
```

Comment:
I find it confusing to have a step 2 in a step 2. can we use a different numbering?

* either non-numericals, `Step a:`
* subnumbering `Step 2.2:`
* or numerate all steps independent of headlines, `Step 3:`